api/v2/routes_database_test.go

```diff
@@ -143,7 +128,35 @@ func Test_API_Routes_Database(t *testing.T) {
 			}
 		})
 	}
-
+	tests = []struct {
+		name string
+		args args
+	}{
+		{"select", args{
+			nil,
+			"select and table", 0, 400,
```

Comment:
shouldn't tests conform 400 error code?

Reply:
it does, the `400` is the code it expects, and the `sendRequest` parameter takes in the expected status code, and verifies it is correct.